diagrams/azure/AI_Machine_Learning.py

```diff
@@ -0,0 +1,35 @@
+# This module is automatically generated by autogen.sh. DO NOT EDIT.
```

Comment:
We use only lowercase for all directories and files. Please make the all sub-directory names in the resource/azure/ lowercase and re-run the autogen script.